akka-http/src/main/resources/reference.conf

```diff
@@ -62,4 +62,56 @@ akka.http {
     # This setting can be enabled to pass those empty events to the application for explicit handling.
     emit-empty-events = off
   }
+
+  # Configuration for the cors directive, does not apply unless the directive is used
+  #cors
+  cors {
```

Comment:
both package names and config are different from akka-http-cors, so there shouldn't be any problems for users that still have akka-http-cors in classpath, right?

Reply:
The problem is what I mentioned in https://github.com/akka/akka-http/pull/4343#discussion_r1453711606 

We have an import-all directives-object/baseclass, now including `cors()`, if the original library is also in scope that will clash and not compile, not entirely sure what happens for the scenario of bumping without recompiling (but the original library does not support newer Akka versions so maybe that is a non issue?)